cylc/rose/__init__.py

```diff
@@ -70,6 +70,26 @@
    for ease of porting Cylc 7 workflows.
 
 
+Special Variables
+-----------------
+
+The Cylc Rose plugin has specific logic for handling a small group of special
+variables in the ``rose-suite.conf`` file:
+
+``ROSE_VERSION``
+   The plugin provides ``ROSE_VERSION`` from your installed Rose Version
+   in the environment section and any templating sections you have defined. If
+   you set ``ROSE_VERSION`` in your ``rose-suite.conf`` it will be replaced.
```

Comment:
The "it will be replaced" part doesn't make it particularly clear what's going on.

Perhaps a deprecation notice explaining that setting this is no longer supported.

```suggestion

   .. deprecated:: 8.0.0
   
      ``[env]ROSE_VERSION``
      
      In  Cylc 7 / Rose 2019 the Rose and Cylc versions could be configured ...
```